cpu/stm32/include/periph/f0/periph_cpu.h

```diff
@@ -23,6 +23,21 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   Starting address of the ROM bootloader
+ *          see application note AN2606
+ */
+#if defined(CPU_LINE_STM32F030x4) || defined(CPU_LINE_STM32F030x6) || \
+    defined(CPU_LINE_STM32F030x8) || defined(CPU_LINE_STM32F031x6) || \
+    defined(CPU_LINE_STM32F051x8)
+#define STM32_LOADER_ADDR   (0x1FFFEC00)
+#elif defined(CPU_LINE_STM32F030xC) || defined(CPU_LINE_STM32F070xB) || \
+      defined(CPU_LINE_STM32F072xB) || defined(CPU_LINE_STM32F091xC)
+#define STM32_LOADER_ADDR   (0x1FFFD800)
```

Comment:
For STM32F071xx/072xx it seems to be `0x1FFFC800`, not `...D800`

For STM32F09xxx and STM32F030xC `0x1FFFD800` is correct, however.

Reply:
Thank you for spotting those!